gigalixir/app.py

```diff
@@ -140,6 +141,11 @@ def ssh_helper(host, app_name, ssh_opts, ssh_cmd, capture_output, *args):
     if len(keys) == 0:
         raise Exception("You don't have any ssh keys yet. See `gigalixir account:ssh_keys:add --help`")
 
+    # use the identity file specified by environment variable
+    id_file = os.environ.get("GIGALIXIR_IDENTITY_FILE")
+    if id_file and not re.match(r'(^|[\s])-i', ssh_opts):
```

Comment:
Will this also match other flags that begin with i? Maybe not because it's a single dash. Should we also check the long version of the flag?

Reply:
in linux/mac, there is only one way to specify the identity file... `-i` and there are no other `-i` based arguments